src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/Inntektsmelding.kt

```diff
@@ -4,14 +4,13 @@ package no.nav.helsearbeidsgiver.inntektsmelding
 
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.UseSerializers
-import kotlinx.serialization.json.JsonObject
 import no.nav.helsearbeidsgiver.utils.json.serializer.LocalDateSerializer
 import no.nav.helsearbeidsgiver.utils.json.serializer.LocalDateTimeSerializer
 import java.time.LocalDateTime
 
 @Serializable
 data class Inntektsmelding(
-    val dokument: JsonObject,
+    val dokument: String,
```

Comment:
Jeg er litt usikker på om denne bør være string for nå, skal sjekke litt på kolonnetype etc hva som er best, men lettere å sammenligne i tester slik i hvert fall